src/tirith/logging.py

```diff
@@ -38,6 +48,14 @@
 
 
 def setup_logging(verbose: bool = False):
+
+    """
+    Set up logging configuration. The default argument verbose: bool = False.
+
+    Here, if `verbose` is set to ``True``,then all the log messages higher than DEBUG level will be displayed in `verbose` format.
+    But, if `verbose` is set to ``False``, all the log messages that are higher than INFO level i.e. ('INFO', 'WARNING', 'CRITICAL') will be dsiplayed in `normal` format.
+    
```

Comment:
Thanks for the detailed docstring! but I think it can be made shorter

I'd write it as the following:
```
    Set up logging configuration.

    :param verbose:  If set to `True` will log all messages higher than DEBUG level, otherwise it will log the messages that are higher than INFO level ('INFO', 'WARNING', 'CRITICAL')
```